tests/test_loads.py

```diff
@@ -68,46 +75,78 @@ def test_loads_pure_gdb_without_crashing():
 
 @pytest.mark.skipif(launched_locally, reason='This test uses binaries compiled on travis builds.')
 def test_loads_binary_without_crashing():
-    output = run_gdb_with_script(binary=BASH_BIN).splitlines()
+    if not os.path.isfile(BINARY):
+        compile_binary(BINARY_SOURCE, BINARY)
+    output = run_gdb_with_script(binary=BINARY).splitlines()
 
-    expected = ['Reading symbols from %s...(no debugging symbols found)...done.' % BASH_BIN]
+    expected = ['Reading symbols from %s...' % BINARY,
+                '(No debugging symbols found in %s)' % BINARY]
     expected += HELLO
 
     assert all(item in output for item in expected)
 
 
 @pytest.mark.skipif(launched_locally, reason='This test uses binaries compiled on travis builds.')
 def test_loads_binary_with_core_without_crashing():
-    output = run_gdb_with_script(binary=BASH_BIN, core=BASH_CORE).splitlines()
+    if not os.path.isfile(BINARY):
+        compile_binary(BINARY_SOURCE, BINARY)
+    if not os.path.isfile(CORE):
+        create_coredump = ['run', f'generate-core-file {CORE}']
+        run_gdb_with_script(binary=BINARY, pyafter=create_coredump)
+        assert os.path.isfile(CORE)
+    output = run_gdb_with_script(binary=BINARY, core=CORE).splitlines()
 
     expected = [
-        'Reading symbols from %s...(no debugging symbols found)...done.' % BASH_BIN,
-        '''[New LWP 13562]''',
-        '''Core was generated by `/home/user/pwndbg/tests/corefiles/bash/binary'.''',
-        '''Program terminated with signal SIGINT, Interrupt.''',
-        '''#0  __addmntent (stream=0x7ffff79a7640 <_nl_global_locale+192>, ''',
-        '''    mnt=<optimized out>) at mntent_r.c:257''',
-        '''257\tmntent_r.c: No such file or directory.'''
+        'Reading symbols from %s...' % BINARY,
+        '(No debugging symbols found in %s)' % BINARY,
+        'Program terminated with signal SIGFPE, Arithmetic exception.',
     ]
     expected += HELLO
 
     assert all(item in output for item in expected)
 
+    # matches for example '[New LWP 45429]'
+    lwp_line = re.compile('^\[New LWP \d+\]$')
+    assert any([lwp_line.match(line) for line in output])
+
+    # matches for example "Core was generated by `./binary'."
+    binary_line = re.compile("^Core was generated by .+$")
+    assert any([binary_line.match(line) for line in output])
+
+    # matches for example '#0  0x0000556e99b7913c in main ()'
+    crash_address_line = re.compile('^#0  0x[0-9a-fA-F]+ in main \(\)$')
+    assert any([crash_address_line.match(line) for line in output])
+
 
 @pytest.mark.skipif(launched_locally, reason='This test uses binaries compiled on travis builds.')
```

Comment:
Now that we compile binaries ourself, I guess we can remove this comment too! :)

Reply:
I included these comments on purpose but I think you are right, these regexes are easy to understand so I'll remove these comments